core/database_api.php

```diff
@@ -37,8 +37,6 @@
 require_api( 'logging_api.php' );
 require_api( 'utility_api.php' );
 
-require_lib( 'adodb/adodb-php/adodb.inc.php' );
```

Comment:
Can you check why this was added from git history?  It may have been added for some upgrade or install path.

Reply:
@vboctor That's just how you implemented the migration from bundled libs to Composer afea93e8c0019f99814b295ff5d23726dac7aa5d

Reply:
>  I will try a new install and uprade with it.

Tried without getting any problem.
@vboctor do you agree to merge?